src/components/editor/ArticleListController.jsx

```diff
@@ -0,0 +1,109 @@
+      slugSearchValue: ""
+    });
+  }
+
+  handleUpdateSlugValue(event) {
+    this.safeSetState({
+      slugSearchValue: event.target.value
+    });
+  }
+
+  handlePageChange(increment, event) {
+    let path = this.props.location.pathname;
+    path = path.split('/');
+    path[3] = (parseInt(path[3])+increment).toString();
+    browserHistory.push(path.join('/'));
+  }
+
+  render() {
+      // For Ling when code reviewing. I was choosing between two style decisions here when making the loading look as I wanted it
+      // I could either do as I did now with quite a few ternary operators. Or I could've put it all in a big if this.state.ready
+      // So that it only processed the data if it had it. But in that case I then just had to put in quite a few var's instead of let's
+      // So they would carry over to the return statement. Which would you think is best?
+      let data = this.state.data ? this.state.data.articlesByTime.paginated : null;
+      let page = this.props.params.page;
+      let maxPage = data ? Math.floor(data.total/pageLength) + (data.total%pageLength > 0) : null;
+      let firstIndex = pageLength*(page-1);
+
+      // If trying to access inacessible page, redirect to page 1
+      if (page < 1 || page > maxPage) {
+        return <p>You have tried accessing a page that doesn't exist. Please press <a href="/articles/page/1/">this link</a> to return to page 1.
+          If you believe this was unintended and there is an error with the website please contact the web development team of The Gazelle.</p>;
+      }
+
+      let indexConverter = data ? [...Array(Math.min(pageLength, data.total-firstIndex-5)).keys()].map((val) => {
+        return val+firstIndex
+      }) : null;
+      let pathStart = "/articles/page/" + page + "/";
+
+    return (
+      <div style={{height: "100%", padding: 0, margin: 0}}>
+        <div style={{height: "100%", width: "50%", float: "left"}}>
+          <h3>Articles</h3>
+          <p style={{width: "75%"}}>Here the 50 most recent articles are shown in the list, if you have an older article in mind,
+          you can find the slug of it either in the Ghost editor, or in URL when you access the website on thegazelle.org
+          and input it below</p>
+          <form onSubmit={this.handleSubmit}>
+            <input type="text" value={this.state.slugSearchValue} placeholder="Input Slug" onChange={this.handleUpdateSlugValue} />
+            <input type="submit" value="Submit" />
+          </form>
+          <br/>
+          {!this.state.ready ? 
+            <p>loading...</p> :
+            <List 
+              data={data}
+              indexConverter={indexConverter}
+              maxHeight="100%"
+              maxWidth="75%"
+              style={{width: "100%", height: "50%", float: "left"}}
+              link={pathStart+"article/slug/"}
+            />
+          }
+          {!this.state.ready ?
+            null :
+            <div style={{width: "75%"}}>
```

Comment:
Use css flexbox, not this 33% width stuff. It gets nasty really fast once you have borders and margins, and padding, cause your width will go over 100% quickly.